components/dashboard/src/components/UsageBasedBillingConfig.tsx

```diff
@@ -214,31 +213,35 @@ export default function UsageBasedBillingConfig({ attributionId }: Props) {
                 )}
                 {showSpinner && (
                     <div className="flex flex-col mt-4 h-52 p-4 rounded-xl bg-gray-50 dark:bg-gray-800">
-                        <div className="uppercase text-sm text-gray-400 dark:text-gray-500">Balance Used</div>
+                        <div className="uppercase text-sm text-gray-400 dark:text-gray-500">Balance</div>
                         <Spinner className="m-2 h-5 w-5 animate-spin" />
                     </div>
                 )}
                 {showBalance && (
                     <div className="flex flex-col mt-4 p-4 rounded-xl bg-gray-50 dark:bg-gray-800">
-                        <div className="uppercase text-sm text-gray-400 dark:text-gray-500">Balance Used</div>
+                        <div className="uppercase text-sm text-gray-400 dark:text-gray-500">Balance</div>
```

Comment:
**question**: What do you think of being more explicit about the balance type to avoid confusion between credits and real money. Also, do we need to also show one or two decimals for accuracy?

| BEFORE | AFTER |
|-|-|
| <img width="696" alt="Frame 546" src="https://user-images.githubusercontent.com/120486/203311407-2635c602-5719-4750-80e8-4e18ce7bf265.png"> | <img width="696" alt="Frame 548" src="https://user-images.githubusercontent.com/120486/203311427-27f65863-5e6b-4a77-ab68-ef89cc20e355.png"> |

Reply:
I like showing "X Credits" better. Also, AFAIR we once decided to not display fractions of credits here.

Reply:
DEAL—Let's come back to this if needed later. 🤝